contrib/qat/compression/qatzip/compressor/test/BUILD

```diff
@@ -0,0 +1,20 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_test",
+    "envoy_contrib_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_contrib_package()
+
+envoy_cc_test(
+    name = "qatzip_compressor_impl_test",
+    srcs = ["qatzip_compressor_impl_test.cc"],
```

Comment:
Probably add this to skip test on arm

``target_compatible_with = envoy_contrib_linux_x86_64_constraints(),``

Reply:
Yeah, I have added the compile option like the test of dlb, it should work.